charts/prometheus-kafka-exporter/Chart.yaml

```
@@ -3,7 +3,7 @@ appVersion: "v1.7.0"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 2.8.0
+version: 2.8.1
```

The minor version number should be raised as a new feature is being added.

```suggestion
version: 2.9.0
```